events/wc_court_events.txt

```diff
@@ -187,6 +187,7 @@ character_event = {
 			can_has_children_trigger = no
 		}
 		NOT = { num_of_dynasty_members = 8 }
+		NOT = { reproduces_asexually }
```

Comment:
reproduces_asexually have no value. It's bug.

Reply:
fixed